api/db-cdr/generate-cdr/make-bq-data-dictionary.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This generates the big query data dictionary tables by importing data from
+# csv at gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary.
+
+set -ex
+
+export BQ_PROJECT=$1  # project
+export BQ_DATASET=$2  # dataset
+export DRY_RUN=$3     # dry run
+
+if [ "$DRY_RUN" == true ]
```

Comment:
I can actually  move the check in for loop below to just print the file thats picked up for loading rather than just checking if there are files.

Reply:
Actually the file names are printed as part of 'gsutil ls', so lists of all the csv files under datadictionary and archive folder are printed out irrespective of dry-run value. 
I think keeping dry_run check here just for errors should be fine then

Reply:
It doesn't matter to me.. but we probably don't need a dry run for this shell script.